### PR TITLE
Zoom In with 'CTRL/CMD + =' issue fixed

### LINKED ISSUE #1031: Ctrl + (+) and Ctrl + (-) are missing form Keyboard shortcuts menu
**Describe the bug**
`Ctrl` + `+` and `Ctrl` + `-` are missing form Keyboard shortcuts menu in the bottom right.

**Expected behavior**
It should tell you about these important shortcuts.

**Desktop (please complete the following information):**
 - Operating System:  Linux Mint 19.3 (Ubuntu 18.04.3)
 - Zulip Desktop Version: 5.4.0

**Additional context**
Zulip is pretty small by standard. A lot of users will want to increase its size by pressing `Ctrl` + `+`

### LINKED ISSUE #1060: Ctrl + = Zooms in by two steps instead of one.
**Describe the bug**

`Ctrl` + `=` Zooms in by two steps instead of one. This can be checked as we need to press `Ctrl` + `-` **twice** in order to go back to the actual(default) size of window.

**To Reproduce**

1) Open Zulip Desktop App
2) Press  `Ctrl` + `=`  or `Ctrl` + `+`

**Desktop (please complete the following information):**
 - Operating System: Ubuntu 18.04 x64
 
 - Zulip Desktop Version: 5.5.0
 
Zoomin

### PR DESCRIPTION
**Pull request by skshamagarwal**
**What's this PR do?**
Closes #1031 & Closes #1060. 
Single zoom-in with both `ControlOrCommand++` and `ControlOrCommand+=`.

**Any background context you want to provide?**
This issue was taking place because the zoomIn function was called twice when we press `ControlOrCommand+=`.
- Called using  accelerators and
- Called while adding global shortcuts

**You have tested this PR on:**
  - [x] Windows
  - [x] Linux/Ubuntu
  - [ ] macOS

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -10800,6 +10801,7 @@
       "version": "1.3.0",
       "resolved": "https://registry.npmjs.org/string_decoder/-/string_decoder-1.3.0.tgz",
       "integrity": "sha512-hkRX8U1WjJFd8LsDJ2yQ/wWWxaopEsABU1XfkM8A+j0+85JAGppt16cr1Whg6KIbb4okU6Mql6BOj+uup/wKeA==",
+      "dev": true,
```
Comment (timabbott) [inline package-lock.json:10804]: Are these changes intentional?

**Comment by timabbott** [comment]
Can you explain in more detail in the commit message your understanding of what the bug is and how this fixes it?  I'd also reference the history of what commit added the line you're editing -- it seems like that `||` block was intentional, and we should understand what was wrong with the original reasoning.

File: package-lock.json (hunk 1 above)
Comment (skshamagarwal) [reply package-lock.json:10804]: No, this one is not intentional

**Comment by skshamagarwal** [comment]
@timabbott Okay will do that. Thanks for reviewing.

**Comment by timabbott** [comment]
@a-saksham can you fix the commit history to clean up the merge commit?  

Check out our [GitHub guide](https://zulip.readthedocs.io/en/latest/git-guide.html) and [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/version-control.html) for more details.

I'm not an expert on this codebase; it'd be better for someone like @akashnimare, @abhigyank, or @andersk to review.

**Comment by andersk** [comment]
My question would be whether the upstream issues that motivated 6e760973ff3e665a798fbbd2e3da83aa740b8c59 and cf96e94470110955da2320a05d3defae0ed52e43 have been fixed by now. If so, we should be able to delete this whole `keydown` listener and restore the `role`s to the menu items.

**Comment by skshamagarwal** [comment]
@andersk I think that we still need that global shortcut keydown section as electron does not use numpad keys as default (I searched a bit, not very sure yet). 
Also, to display the shortcut keys properly in the submenus, probably accelerators will be much better than the role in case for our zoomIn functionality

**Comment by skshamagarwal** [comment]
@

> @a-saksham can you fix the commit history to clean up the merge commit?
> 
> Check out our [GitHub guide](https://zulip.readthedocs.io/en/latest/git-guide.html) and [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/version-control.html) for more details.
> 
> I'm not an expert on this codebase; it'd be better for someone like @akashnimare, @abhigyank, or @andersk to review.

Yes sure, I will clean the commit history.

**Comment by andersk** [comment]
This does not appear to address #1031. I committed a different fix for #1060 as bd0869ec07320653a5b50d3ad0ebbb832be0ab2b.
